.kitchen.yml

```diff
@@ -0,0 +1,36 @@
+---
+driver:
+  name: vagrant
+
+provisioner:
+  name: chef_solo
+
+platforms:
+# Loop through two lists and output a total matrix of all possible platform + chef versions
+<% %w(
+      ubuntu-14.04
+      ubuntu-12.04
+      ubuntu-10.04
+      centos-7.0
+      centos-6.6
+      centos-5.11
+    ).each do |platform_version| %>
+<% %w(
+      11.16.4
+      11.0.0
```

Comment:
More work is needed before adding 12 :p
Will work on it after this is merged so there is more thorough testing